File: csharpfunctions/API/API_1.cs
using Microsoft.AspNetCore.Mvc;
using Microsoft.Azure.WebJobs;
using Microsoft.Azure.WebJobs.Extensions.Http;
using Microsoft.AspNetCore.Http;
using Microsoft.Extensions.Logging;

namespace csharpfunctions
{
    public static class API_1
    {
        [FunctionName("API_1")]
        public static IActionResult Run(
            [HttpTrigger(AuthorizationLevel.Function, "get", "post", Route = null)] HttpRequest req,
            ILogger log)
        {
            log.LogInformation("C# API 1 function processed a request.");
            return new OkObjectResult("Hello from C#!");
        }
    }
}

File: fsharpfunctions/API/API_1.fs
namespace FSharpFunctions

open Microsoft.AspNetCore.Mvc
open Microsoft.Azure.WebJobs
open Microsoft.Azure.WebJobs.Extensions.Http
open Microsoft.AspNetCore.Http
open Microsoft.Extensions.Logging

module API_1 =
    [<FunctionName("API_1")>]
    let run ([<HttpTrigger(AuthorizationLevel.Function, "get", "post", Route = null)>]req: HttpRequest) (log: ILogger) =
        log.LogInformation("F# API 1 function processed a request.")
        OkObjectResult("Hello from F#!")